src/views/tx/details/index.js

```diff
@@ -67,9 +70,12 @@ class TxDetails extends React.Component {
                 </div>
 
                 <div className="one-table-container white" style={{marginTop: "70px"}}>
-                    <EventTable
-                        events={data.tx_result.events}
-                    />
+                  {
+                    data.tx_result.events === null && <TransactionMessage msg={msg} />
+                  }
+                  {
+                    data.tx_result.events !== null && <EventTable events={data.tx_result.events} />
```

Comment:
This is just in case someone goes back in time and tries to read a transaction that was issued prior to the structure change, meaning issued at a time when events were still populated.